src/Calculator/App.xaml.cpp

```diff
@@ -70,6 +70,33 @@ App::App()
     this->HighContrastAdjustment = ApplicationHighContrastAdjustment::None;
 
     this->Suspending += ref new SuspendingEventHandler(this, &App::OnSuspending);
+    auto value = Windows::Storage::ApplicationData::Current->LocalSettings->Values->Lookup(L"themeSetting");
+    auto DefaultTheme = ref new Windows::UI::ViewManagement::UISettings();
+    auto uiTheme = DefaultTheme->GetColorValue(Windows::UI::ViewManagement::UIColorType::Background).ToString();
+    if (value != nullptr)
+    {
+        String ^ colorSetting = safe_cast<String ^>(value);
```

Comment:
Could we use an enum here instead of a string?

Reply:
@joseartrivera Why do we need to do that? Thanks for your suggestions, by the way.

Reply:
I left this comment again, didn't realize I had left it previously. An enum is safter to use than a string, it avoids people from entering a typo into the string if they make changes to this in the future.